png-resizer/app/lib/FutureEither.scala

```diff
@@ -0,0 +1,47 @@
+package lib
+
+import common.StopWatch
+import grizzled.slf4j.Logging
+import lib.FutureEither._
+import metrics.FrontendTimingMetric
+import play.api.mvc.Result
+
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.Future
+import scalaz.EitherT._
+import scalaz.{EitherT, Monad, \/-}
+
+object FutureEither {
+
+  implicit val futureMonad = new Monad[Future] {
```

Comment:
Ah never mind, it looks like they removed it https://github.com/typelevel/scalaz-outlaws/blob/master/src/main/scala/scalaz/outlaws/std/Future.scala